docs/plugins/replaygain.rst

```diff
@@ -10,10 +10,10 @@ playback levels.
 Installation
 ------------
 
-This plugin can use one of three backends to compute the ReplayGain values:
-GStreamer, mp3gain (and its cousin, aacgain), Python Audio Tools. mp3gain
-can be easier to install but GStreamer and Audio Tools support more audio
-formats.
+This plugin can use one of many backends to compute the ReplayGain values:
+GStreamer, mp3gain (and its cousin, aacgain), Python Audio Tools or ffmpeg.
+mp3gain can be easier to install but GStreamer, Audio Tools and ffmpeg support
+more audio formats.
```

Comment:
I would even say that ffmpeg belongs to the "easy to install" camp! It's available in nearly every package manager ever, and it usually comes as a complete package with everything included.